lib/assembly-image/image.rb

```diff
@@ -9,11 +9,13 @@ module Assembly
   class Image < Assembly::ObjectFile
     # @return [integer] image height in pixels
     def height
+      # use exif to avoid problems with libvips in circleci not speaking jp2
```

Comment:
Would this imply we don't have a working test for jp2 as a source image on Circle?  Should we work to resolve that problem?

Reply:
We do have a working test for jp2 as a source image - see https://github.com/sul-dlss/assembly-image/blob/main/spec/image_spec.rb#L241.  It's not a great test, but it at least shows expected behavior.

Reply:
@ndushay I don't think the `:input_path` is set to a JP2 in that context.

Reply:
I'll make a separate ticket to fix the jp2 test so this PR can go through.

Reply:
https://github.com/sul-dlss/assembly-image/issues/103